src/routes/student/Shop.tsx

```diff
@@ -9,6 +9,8 @@ import { getHairItems, getPantsItems, getShirtItems, getShoesItems } from '../..
 import { ItemCard } from '../../components/student/ItemCard'
```

Comment:
From my testing it looks like wood background isn't showing up for the hair panel?

Reply:
it's because we did "backgroundSize: 'cover'" instead of "contain" - I think the background somehow stretched to fill the container because there are many many more hair cards than of other items